conference/talks.py

```diff
@@ -0,0 +1,61 @@
+from django.conf.urls import url
+from django.shortcuts import get_object_or_404
+from django.template.response import TemplateResponse
+
+from conference.models import Talk
+
+
+def talk(request, talk_slug):
+    """
+    Display Talk
+    """
+    talk = get_object_or_404(Talk, slug=talk_slug)
```

Comment:
Should this page be accessible for all talks or only for approved talks?

Reply:
Not yet, but we might temporarily put @staff_member_required on it – context: Program WG is using it this weekend, before things get approved.